README.md

```diff
@@ -307,9 +309,9 @@ On the next round, the Client checks each block against the filter, and begins a
 
 ### 3.4.1 Indexing
 
-Indexes MUST be generated from hashes using the following strategy, based on whether or not the filter fits perfectly into a power of two ($2^c$). This is a single algorithm, but if the size of the Bloom filter is a power of two, rejection sampling MAY be omitted. Using a Bloom filter that is a power of two is RECOMMENDED since it avoids resampling.
+Indexes MUST be generated from hashes using the following strategy, based on whether or not the filter fits perfectly into a power of two ($2^c$). This is a single algorithm, but if the size of the Bloom filter is a power of two, rejection sampling isn't needed. Because of that, using a Bloom filter that is a power of two is RECOMMENDED.
 
-NB: The Bloom filter's bucket order (index) MUST be interpreted big-endian and zero-indexed. All hashes generated for indexing MUST be interpreted as big-endian natural numbers.
+NB: The Bloom filter's bucket order (index) MUST be interpreted zero-indexed.
```

Comment:
The XXHash hashes are actually never serialized. They come out of the library as ... `u64`s.
XXHash has some recommendations on serializing these `u64`s as big-endian, but that's irrelevant for our purposes.